Core/GraphQL/GraphQLFiles/sell_fragment_product.graphql

```diff
@@ -0,0 +1,59 @@
+fragment FragmentProduct on Product {
+    id
+    status
+    reference
+    externalId
+    images {
+      imageUrl
+      position
+    }
+    modifierLists {
+      id
+      status
+      externalId
+      reference
+      multipleSelect
+      mandatorySelect
+      maxAllowed
+      items {
```

Comment:
Can this be `FragmentProductModifierItem`

Reply:
just saw these